nilearn/plotting/surf_plotting.py

```diff
@@ -66,6 +66,14 @@
         "anterior": (0, 90),
         "posterior": (0, 270),
     },
+    "both": {
+        "lateral": (0, 0),
+        "medial": (0, 180),  # does not make sense, will show left hemisphere
```

Comment:
do we want to change those to left and right?

Reply:
yes good idea

Reply:
Thought about it and it seems when plotting both hemis we must replace:
    - lateral -> right
    - medial -> left